pages/common/ksh.md

```diff
@@ -0,0 +1,20 @@
+# ksh
+
+> Korn Shell.
+> `bash` and `sh`-compatible command line interpreter.
+
+- Start interactive command line interpreter:
+
+`ksh`
+
+- Execute command passed as parameter:
```

Comment:
Execute _a_ command. No need for the rest. The bash page also keeps it like that.